db/migrate/20170707205938_default_value_for_reduction_subgroup.rb

```diff
@@ -0,0 +1,6 @@
+class DefaultValueForReductionSubgroup < ActiveRecord::Migration[5.1]
+  def change
+    change_column :reductions, :subgroup, :string, :default => "default"
```

Comment:
yeah, i changed my mind about this part-way through and didn't go far enough back. good catch!